src/api/starWars-utils.ts

```diff
@@ -1,10 +1,8 @@
-import { RowCharacter, ModifiedCharacter, CharactersBySpecies } from '../types'
-
-const calcBMI = (height: number, mass: number): number => mass / Math.pow(height, 2)
+import { RowCharacter, ModifiedCharacter, CharactersBySpecies } from './starWars-types'
 
 export const modifyRowCharacters = (characters: RowCharacter[]): ModifiedCharacter[] => {
   const modifiedCharacters: ModifiedCharacter[] = []
-  console.log(characters, 'characters')
+  const calcBMI = (height: number, mass: number): number => mass / Math.pow(height, 2)
```

Comment:
Just nitpicking, you can omit the return type because of Type Inference.

`const calcBMI = (height: number, mass: number) => mass / Math.pow(height, 2)`
It should work👍